From 884017d24f554a908a75ce1aaf490957e89e2b34 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sat, 28 Sep 2024 11:57:40 +0100
Subject: [PATCH 1/1] packagekit: Set PK_INFO_ENUM_SECURITY as
 AS_URGENCY_KIND_CRITICAL

This ensures security updates from packagekit get marked appropriately
in the UI.

Follow up to #1676.
---
 plugins/packagekit/gs-plugin-packagekit.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/plugins/packagekit/gs-plugin-packagekit.c b/plugins/packagekit/gs-plugin-packagekit.c
index a93622128..577363687 100644
--- a/plugins/packagekit/gs-plugin-packagekit.c
+++ b/plugins/packagekit/gs-plugin-packagekit.c
@@ -3077,6 +3077,9 @@ get_updates_cb (GObject      *source_object,
 		case PK_INFO_ENUM_IMPORTANT:
 			gs_app_set_update_urgency (app, AS_URGENCY_KIND_HIGH);
 			break;
+		case PK_INFO_ENUM_SECURITY:
+		    gs_app_set_update_urgency (app, AS_URGENCY_KIND_CRITICAL);
+			break;
 		case PK_INFO_ENUM_CRITICAL:
 			gs_app_set_update_urgency (app, AS_URGENCY_KIND_CRITICAL);
 			break;
-- 
2.46.1

